docs/geozarr-eopf-specification.qmd

```diff
@@ -0,0 +1,304 @@
+---
+title: "GeoZarr for EOPF - Experimental Specification"
+subtitle: "Transforming Earth Observation Processing Framework Zarr to GeoZarr V3"
+author: "GeoZarr Examples Project"
+date: today
+format:
+  html:
+    toc: true
+    toc-depth: 3
+    code-fold: false
+    code-tools: true
+---
+
+## Overview
+
+This document outlines the experimental GeoZarr specification for transforming EOPF (Earth Observation Processing Framework) Zarr stores into GeoZarr V3 compliant datasets. The specification is based on the implementation in `06_multiscales_as_WebMercatorQuad_EOPFZarrV3.ipynb` and demonstrates how GeoZarr could be adapted for Zarr format 3 while maintaining compatibility with existing geospatial tools.
+
+GeoZarr V3 extends the Zarr V3 format to support geospatial data with proper coordinate reference systems, multiscale overviews, and CF-compliant metadata, specifically addressing the gaps found in EOPF Zarr format.
+
+## Gaps in EOPF Zarr Format
+
+The current EOPF Zarr stores have several gaps that prevent them from being fully GeoZarr compliant:
+
+### Issues Identified
+
+| Issue | Description | Impact |
+|-------|-------------|---------|
+| **Missing CF Standard Names** | Variables lack `standard_name` attributes | Reduces interoperability with CF-compliant tools |
+| **Incomplete CRS Information** | CRS stored only as `proj:epsg` attribute | Limited compatibility with rioxarray and geospatial tools |
+| **Missing Grid Mapping** | No `grid_mapping` attribute linking to spatial reference | Geospatial tools can't detect coordinate system |
+| **No Multiscale Support** | Lacks overview levels and multiscale metadata | Poor performance for multi-scale visualization |
+| **Missing Coordinate Arrays** | Overview levels lack proper x/y coordinate arrays | Cannot perform spatial operations on overview data |
```

Comment:
For data that can be described by an affine transformation (e.g., regular grids), the coordinate arrays should not be required because they increase storage space, increase data transfer/load times, and can introduce errors associated with numerical precision relative to a functional coordinates. Geospatial processing libraries can produce explicit coordinate arrays if needed based on the affine transformation.